server/coordinator_test.go

```diff
@@ -801,22 +803,28 @@ func (s *testOperatorControllerSuite) TestStoreOverloaded(c *C) {
 	c.Assert(tc.addRegionStore(2, 40), IsNil)
 	c.Assert(tc.addRegionStore(1, 10), IsNil)
 	c.Assert(tc.addLeaderRegion(1, 2, 3, 4), IsNil)
+	start := time.Now()
 	op1 := lb.Schedule(tc)[0]
 	c.Assert(op1, NotNil)
 	c.Assert(oc.AddOperator(op1), IsTrue)
 	for i := 0; i < 10; i++ {
+		if time.Now().Sub(start) >= time.Second*9/10 {
+			break
+		}
 		c.Assert(lb.Schedule(tc), IsNil)
 	}
 	c.Assert(oc.RemoveOperator(op1), IsTrue)
-	time.Sleep(1 * time.Second)
+	time.Sleep(time.Duration(60/cfg.StoreBalanceRate) * time.Second)
```

Comment:
It's too long to wait for `6s`, maybe 2 s?

Reply:
I have tried 2s,3s,5s, but failed.

Reply:
why failed? In theory, only one second is enought.

Reply:
Maybe because I used `60/cfg.StoreBalanceRate` to adjust the sleep time, it is now changed to 2 seconds.